CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribution Guidelines
+We welcome feedback, bug reports, and pull requests!
+For pull requests (PRs), please stick to the following guidelines:
```

Comment:
Maybe we should put a part in about discussions actually

```suggestion
## Discussions

If you're unsure about anything, feel free to ask questions by opening a discussion or reaching out to us in the issues section.

## Pull Requests
For pull requests (PRs), please stick to the following guidelines:
```

Reply:
Yes please